GoogleSignIn/Sources/GIDDataFetcher/Implementations/GIDDataFetcher.m

```diff
@@ -0,0 +1,40 @@
+#import "GoogleSignIn/Sources/GIDDataFetcher/Implementations/GIDDataFetcher.h"
+
+#ifdef SWIFT_PACKAGE
+@import AppAuth;
+@import GTMAppAuth;
+#else
+#import <AppAuth/AppAuth.h>
+#import <GTMAppAuth/GTMAppAuth.h>
+#endif
+
+NS_ASSUME_NONNULL_BEGIN
+
+// Maximum retry interval in seconds for the fetcher.
+static const NSTimeInterval kFetcherMaxRetryInterval = 15.0;
+
+@implementation GIDDataFetcher
+
+- (void)startFetchURL:(NSURL *)URL
+    fromAuthState:(OIDAuthState *)authState
+      withComment:(NSString *)comment
+       completion:(void (^)(NSData *, NSError *))completion {
```

Comment:
Same here, align the colons

Reply:
Shouldn't these be aligned on the `startFetchURL` colon?

Reply:
Yes.